README.md

```diff
@@ -49,19 +49,15 @@ To enable it, you must either set the USE_REAL_AWS variable or specify a
 real openshift-ansible image to use in the ANSIBLE_IMAGE variable.
 
   * `USE_REAL_AWS=1 contrib/examples/create-cluster.sh`
-  * `ANSIBLE_IMAGE=openshift/origin-ansible:latest contrib/examples/create-cluster.sh`
+  * `ANSIBLE_IMAGE=cluster-operator-canary:latest contrib/examples/create-cluster.sh`
```

Comment:
This should be `cluster-operator-ansible:canary` right?

Reply:
Good catch, fixed now.